src/Elm/Ast.hs

```diff
@@ -70,7 +72,7 @@ isEnum ElmType{..} = null elmTypeVars && null (foldMap elmConstructorFields elmT
 getConstructorNames :: ElmType -> [Text]
 getConstructorNames ElmType{..} = map elmConstructorName $ toList elmTypeConstructors
 
--- | Primitive elm types; hardcoded by the language.
+-- | Primitive elm types which are parts of a language
```

Comment:
this comment was untrue. not all things that were defined as part of prim were "hardcoded" in the language.